src/angular-business/notification-simple/notification-simple-container/notification-simple-container.component.scss

```diff
@@ -0,0 +1,140 @@
+$sbbBusiness: true;
+@import '../../../angular-core/styles/common';
+
+$notificationHeight: 51;
+$notificationBaseWidth: 100%;
+$notificationIconWidth: 24;
+$notificationIconHeight: 24;
+$notificationDefaultBackgroundColor: $sbbColorWhite;
+$notificationErrorBackgroundColor: $sbbColorRed;
+$notificationWarnBackgroundColor: $sbbColorOrange;
+$notificationLinksHeight: 72;
+$notificationBorderRadius: 2;
+$notificationDefaultFontSize: $sizeFontDefault;
+$notificationIconMarginLeft: 16;
+$notificationIconMarginRight: 22;
+$notificationMessageMarginLeft: 8;
+$notificationMessageMarginRight: 24;
+$notificationToastOffset: 20;
+$notificationToastMaxWidth: 251;
+$notificationVerticalMargin: 12;
+
+.sbb-notification {
```

Comment:
I made the notification-container component refer to the notification.component.scss stylesheet, though I'm not sure if that's what you meant by "reuse". Since the styles are identical it does seem to make sense to use only one stylesheet though.